src/System.Collections.NonGeneric/src/packages.config

```diff
@@ -1,14 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="System.Collections" version="4.0.10-beta-22512" />
-  <package id="System.Diagnostics.Contracts" version="4.0.0-beta-22512" targetFramework="portable-net45+win" />
-  <package id="System.Diagnostics.Debug" version="4.0.10-beta-22512" targetFramework="portable-net45+win" />
-  <package id="System.Diagnostics.Tools" version="4.0.0-beta-22512" targetFramework="portable-net45+win" />
-  <package id="System.Globalization" version="4.0.10-beta-22512" />
-  <package id="System.Reflection" version="4.0.10-beta-22512" targetFramework="portable-net45+win" />
-  <package id="System.Reflection.Primitives" version="4.0.0-beta-22512" targetFramework="portable-net45+win" />
-  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22512" />
-  <package id="System.Runtime" version="4.0.20-beta-22512" />
-  <package id="System.Runtime.Extensions" version="4.0.10-beta-22512" />
-  <package id="System.Threading" version="4.0.0-beta-22512" />
+  <package id="System.Collections" version="4.0.10-beta-22703" />
+  <package id="System.Diagnostics.Contracts" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Diagnostics.Debug" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Diagnostics.Tools" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Globalization" version="4.0.10-beta-22703" />
+  <package id="System.Reflection" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Reflection.Primitives" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Resources.ResourceManager" version="4.0.0-beta-22703" />
+  <package id="System.Runtime" version="4.0.20-beta-22703" />
+  <package id="System.Runtime.Extensions" version="4.0.10-beta-22703" />
+  <package id="System.Threading" version="4.0.10-beta-22703" />
```

Comment:
Intentional. Only newest version for now. We will have a story for referencing 4.0.0 before long.